addons/hr_fleet/models/vehicle_assignation_log.py

```diff
@@ -0,0 +1,25 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from odoo import api, fields, models
+
+
+class FleetVehicleAssignationLog(models.Model):
```

Comment:
I know that the model name is not sexy, but according to the guidelines, the file should be named ``fleet_vehicle_assignation_log``